curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606c3fd5634684d48a7887b.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 6606c3fd5634684d48a7887b
+title: Step 32
+challengeType: 20
+dashedName: step-32
+---
+
+# --description--
+
+This expression generates a list of boolean values representing whether the condition `self.board[row][col] != num` is `True` or `False` for each element in the specified column across all rows.
```

Comment:
Which expression?

Reply:
Maybe it's a bit vague but it refers to the generator expression in the editable region
`(self.board[row][col] != num for row in range(9))`